service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/EpisodeOfCareMapper.java

```diff
@@ -32,10 +32,10 @@
 
     public Optional<EpisodeOfCare> mapEpisodeOfCare(POCDMT000002UK01ClinicalDocument1 clinicalDocument, Reference subject) {
         POCDMT000002UK01EncompassingEncounter encompassingEncounter = clinicalDocument.getComponentOf()
-            .getEncompassingEncounter();
+                .getEncompassingEncounter();
 
         if (encompassingEncounter.isSetResponsibleParty() &&
-            encompassingEncounter.getResponsibleParty().getAssignedEntity() != null) {
+                encompassingEncounter.getResponsibleParty().getAssignedEntity() != null) {
```

Comment:
please remove additional spacing here

Reply:
This is the code formatter, not added intentionally